manifests/0000_90_dns-operator_03_prometheusrules.yaml

```diff
@@ -34,3 +34,15 @@ spec:
           severity: warning
         annotations:
           message: "CoreDNS is returning SERVFAIL for {{ $value | humanizePercentage }} of requests."
+      - alert: CoreDNSForwardHealthcheckFailureCount
+        annotations:
+          message: '{{ $value }} A CoreDNS counter for the number of failed health checks for an upstream server on {{ $labels.instance }}'
+        expr: coredns_forward_healthcheck_failure_count_total > 0
+        labels:
+          severity: warning
+      - alert: CoreDNSForwardHealthcheckBrokenCount
+        annotations:
+          message: '{{ $value }} A CoreDNS counter for the number of health check failures for all upstream servers on {{ $labels.instance }}'
+        expr: coredns_forward_healthcheck_broken_count_total > 0
```

Comment:
Same for this one as above. Note this is a critical severity alert you will page people during the night to fix something, so we want to make sure this alert is correct. You also need to add a for, so the time you want this to be true (the error based alert) for at least 5, 10 or 15 minutes before this fires. (see examples above)